### DISCUSSION TITLE
Docs don't work in Chrome

### DISCUSSION THREAD

**Discussion by lroos** [microsoft/fluentui#28547] [General] [answered]
Is it just me or when you visit https://react.fluentui.dev/ in Chrome is there no navigation sidebar? I just see the Concepts/Introduction page with a big "9" and not much else. I almost dismissed this as a complete lack of documentation because nothing was accessible.

<img width="363" alt="image" src="https://github.com/microsoft/fluentui/assets/1506230/0f1aac11-0a87-4c76-b32d-d0586d9fa5f3">

**Comment by khmakoto** [comment]
@lroos the navigation bar appears for me in Google Chrome. If you have by any chance resized your browser it may be that the navigation bar is collapsed, in that case, there should be a `sidebar` button to open it available at the bottom of the viewport.

**Reply by lroos** [reply, marked as answer]
Oh... it reappeared when I pressed 'F'. Seems I had set storybook to fullscreen once upon a time and that persisted😅. There was no indication that it was still in full screen mode when I navigated back to the site.
